lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -1440,6 +1451,30 @@ module Resources
         'op': 'remove',
         'path': '/relationship/0'
       },
+      {
+        'op': 'replace',
+        'path': '/relationship/0/extension',
+        'value': [
+          {
+            'valuePeriod': {
+              'start': '2016-12-30T22:26:08.000Z',
+              'end': '2036-12-30T22:26:08.000Z'
```

Comment:
Should the values for the period extension be the data from the C1941 or S1941 env or something similar to them?
@alexbeckner1

Reply:
I don't think it really matters on the PATCH examples as long as someone could potentially use those values in those environment.